llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -13912,10 +13918,11 @@ bool SLPVectorizerPass::vectorizeStores(ArrayRef<StoreInst *> Stores,
       unsigned MinVF = TTI->getStoreMinimumVF(
           R.getMinVF(DL->getTypeSizeInBits(ValueTy)), StoreTy, ValueTy);
 
-      if (MaxVF <= MinVF) {
+      if (MaxVF < MinVF) {
```

Comment:
I mean this change increases cases, where the vectorization may be triggered, since you're loosening the restriction. So need to add the test for such a case.

Reply:
I think I might be missing something here. The original code only guarded the debug output by this check AFAICT, but always also reached the for loop below, which would not be entered if `MaxVF < MinVF`. 

Now the code continues early instead of reaching the for loop, so it should only restrict the cases where we execute the loop below (and could vectorize)?

Reply:
Ah, Yes, missed there were continue before.